dispatcher/src/main/kotlin/pl/droidsonroids/testing/mockwebserver/condition/Condition.kt

```diff
@@ -1,15 +1,16 @@
 package pl.droidsonroids.testing.mockwebserver.condition
 
-import okhttp3.HttpUrl
+import okhttp3.mockwebserver.RecordedRequest
 
 /**
  * Represents set of mocked request properties which have to match received request.
  * Conditions are checked by <code>FixtureDispatcher</code> in order defined by values returned from <code>compareTo</code>.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Line detected that is longer than the defined maximum line length in the code style. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.droidsonroids.pl/coding_rules#rule_key=detekt-kotlin%3AMaxLineLength)